data/hails.txt

```diff
@@ -344,6 +356,8 @@ phrase "friendly civilian"
 		". "
 	word
 		"Their music"
+		"Their taste"
```

Comment:
This doesn't seem to fit very well with most of the things that could come next.

Reply:
Their taste was removed, as I do agree with you on this one.

Reply:
I don't see any new commits.